.npmignore

```diff
@@ -0,0 +1,7 @@
+node_modules/
+.nyc_output/
+coverage/
+src/
```

Comment:
That is an absolutely valid point, thank's for bringing that on the table. I'll change that back soon.